client/my-sites/email/email-management/email-home.tsx

```diff
@@ -0,0 +1,198 @@
+import EmptyContent from 'calypso/components/empty-content';
+import Main from 'calypso/components/main';
+import SectionHeader from 'calypso/components/section-header';
+import { useEmailsManagementDomainsQuery } from 'calypso/data/emails/use-emails-management-domains';
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
+import { hasGSuiteWithUs } from 'calypso/lib/gsuite';
+import { hasTitanMailWithUs } from 'calypso/lib/titan';
+import EmailHeader from 'calypso/my-sites/email/email-header';
+import EmailListActive from 'calypso/my-sites/email/email-management/home/email-list-active';
+import EmailListInactive from 'calypso/my-sites/email/email-management/home/email-list-inactive';
+import EmailNoDomain from 'calypso/my-sites/email/email-management/home/email-no-domain';
+import EmailPlan from 'calypso/my-sites/email/email-management/home/email-plan';
+import EmailProvidersComparisonStacked from 'calypso/my-sites/email/email-providers-stacked-comparison';
+import { emailManagementTitanSetUpMailbox } from 'calypso/my-sites/email/paths';
+import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import hasLoadedSites from 'calypso/state/selectors/has-loaded-sites';
+import { createSiteDomainObject } from 'calypso/state/sites/domains/assembler';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { ResponseDomain } from 'calypso/lib/domains/types';
+import type { TranslateResult } from 'i18n-calypso';
+import type { ReactElement } from 'react';
+
+import './style.scss';
+
+interface EmailManagementHomeProps {
+	emailListInactiveHeader: ReactElement;
+	sectionHeaderLabel: TranslateResult;
+	selectedDomainName: string;
+	showActiveDomainList?: boolean;
+	source: string;
+}
+
+function toTitleCase( words: string[] | string ): string {
+	if ( typeof words === 'string' ) {
+		words = words.trim().split( ' ' );
+	}
+
+	const result = words.map( function ( word ) {
+		return word.charAt( 0 ).toUpperCase() + word.slice( 1 );
+	} );
+
+	return result.join( ' ' );
+}
+
+const EmailManagementHome = ( props: EmailManagementHomeProps ): ReactElement => {
+	const {
+		emailListInactiveHeader,
+		showActiveDomainList = true,
+		selectedDomainName,
+		sectionHeaderLabel,
+		source,
+	} = props;
+
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+	const canManageSite = useSelector( ( state ) =>
+		canCurrentUser( state, selectedSite?.ID ?? 0, 'manage_options' )
+	);
+	const currentRoute = useSelector( ( state ) => getCurrentRoute( state ) );
+	const hasSitesLoaded = useSelector( ( state ) => hasLoadedSites( state ) );
+
+	const { data, isFetched } = useEmailsManagementDomainsQuery( selectedSite?.ID ?? 0, {
+		retry: false,
+	} );
+
+	const hasSiteDomainsLoaded = isFetched;
```

Comment:
We could simplify this using an alias:

```suggestion
	const { data, isFetched: hasSiteDomainsLoaded } = useEmailsManagementDomainsQuery( selectedSite?.ID ?? 0, {
		retry: false,
	} );
```